admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node can reserve a portion of its resources for use by all underlying
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
+components (e.g., *sshd*, *NetworkManager*) on a node host. Once specified, the
+scheduler has more information about the resources (e.g., memory, CPU) a node
+has allocated for pods.
+
+[[allocating-node-settings]]
+== Configuring Nodes for Allocated Resources
+
+Resources reserved for node components are based on two node settings:
+
+[options="header",cols="1,2"]
+|===
+
+|Setting |Description
+
+|`*kube-reserved*`
+| A set of `<resource_type>=<resource_quantity>` pairs (e.g.,
```

Comment:
Changed this to keep with normal doc styling. Hopefully it was OK to change this from the original `ResourceName=ResourceQuantity`, LMK if it really needs to be in that form for some reason.